app/src/shared/services/specmath.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { SpecOperation, FilesRequestBody, SpecMathResponse } from '../interfaces';
+import { routes } from '../routes';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+const requestOptions = {
+  headers: new HttpHeaders({
+    'content-type': 'application/json',
+  }),
+};
+
+@Injectable()
+export class SpecMathService {
+  constructor(private http: HttpClient) {
+
+  };
+
+  processFiles(spec1: string, spec2: string, operation: SpecOperation, driverFile?: string): Observable<SpecMathResponse> {
```

Comment:
Suggestion (for sometime soon) - get an auto formatter setup so you don't end up with long lines like this.

Reply:
Also - wouldn't be a bad idea to add tsdoc summaries to the service call(s)

Reply:
I added a rule on the linter for line length and a TSdoc summary